### PR TITLE
Unify the msg of 'Memory exceed limit'

### PR DESCRIPTION
**Pull request by EmmyMiao87**
The new msg of limit exceed: "Memory exceed limit. %msg, Backend:%ip, fragment:%id Used:% , Limit:%. xxx".
This commit unifies the msg of 'Memory exceed limit' such as check_query_state, RETURN_IF_LIMIT_EXCEEDED and LIMIT_EXCEEDED.

### CODE REVIEW

File: be/src/exec/sort_node.cpp [hunk 1]
```diff
@@ -146,8 +146,9 @@ Status SortNode::sort_input(RuntimeState* state) {
         RETURN_IF_ERROR(child(0)->get_next(state, &batch, &eos));
         RETURN_IF_ERROR(_sorter->add_batch(&batch));
         RETURN_IF_CANCELLED(state);
-        RETURN_IF_LIMIT_EXCEEDED(state, 
-                "Sort node was getting next from the child.");
+        std::stringstream msg;
+        msg << "Sort, while getting next from the child.";
+        RETURN_IF_LIMIT_EXCEEDED(state, msg.str());
```
Comment (imay) [inline be/src/exec/sort_node.cpp]:
```suggestion
        RETURN_IF_LIMIT_EXCEEDED(state, "Sort, while getting next from the child.");
```

File: be/src/exec/partitioned_aggregation_node.cc [hunk 2]
```diff
@@ -232,8 +232,9 @@ Status PartitionedAggregationNode::open(RuntimeState* state) {
     bool eos = false;
     do {
         RETURN_IF_CANCELLED(state);
-        // RETURN_IF_ERROR(QueryMaintenance(state));
-        RETURN_IF_ERROR(state->check_query_state());
+        std::stringstream msg;
+        msg << "Partitioned aggregation, while getting next from child 0.";
+        RETURN_IF_ERROR(state->check_query_state(msg.str()));
```
Comment (imay) [inline be/src/exec/partitioned_aggregation_node.cc]:
```suggestion
        RETURN_IF_ERROR(state->check_query_state("Partitioned aggregation, while getting next from child 0."));
```

File: be/src/runtime/plan_fragment_executor.cpp [hunk 3]
```diff
@@ -514,7 +514,7 @@ void PlanFragmentExecutor::update_status(const Status& status) {
 
         if (_status.ok()) {
             if (status.is_mem_limit_exceeded()) {
-                _runtime_state->set_mem_limit_exceeded();
+                _runtime_state->set_mem_limit_exceeded(status.get_error_msg());
```
Comment (imay) [inline be/src/runtime/plan_fragment_executor.cpp]:
```suggestion
                _runtime_state->set_mem_limit_exceeded(status.to_string());
```

File: be/src/runtime/buffered_block_mgr.cpp [hunk 4]
```diff
@@ -76,9 +77,9 @@ Status BufferedBlockMgr::get_new_block(Block** block, int64_t len) {
     new_block->_buffer_desc->block = new_block;
     *block = new_block;
 
-    if (UNLIKELY(_state->instance_mem_tracker()->any_limit_exceeded())) {
-        return Status::MemoryLimitExceeded("Memory limit exceeded");
-    }
+    std::stringstream msg;
+    msg << "Buffered block mgr, while getting new block";
+    RETURN_IF_LIMIT_EXCEEDED(_state, msg.str());
```
Comment (imay) [inline be/src/runtime/buffered_block_mgr.cpp]: No need to use stringstream

**Review by imay — CHANGES_REQUESTED** [review]

File: be/src/runtime/plan_fragment_executor.cpp (hunk 3 above)
Comment (EmmyMiao87) [reply be/src/runtime/plan_fragment_executor.cpp]: `set_mem_limit_exceeded ` already set the code of `MemoryLimitExceeded`

**Review by imay — APPROVED** [review]

**Pull request merged**